test/module/maintenance/test_patch_spec.py

```diff
@@ -39,6 +39,12 @@ def setUp(self):
     def test_success_rds_dbcluster(self):
         """Success test"""
         patched = patch_spec(self.spec, 'us-east-1')
+        self.assertTrue(
+            patched['PropertyTypes']['AWS::CloudFront::Distribution.DistributionConfig']['Properties']['DefaultCacheBehavior']['Required'])
+        self.assertTrue(
+            patched['PropertyTypes']['AWS::CloudFront::Distribution.DistributionConfig']['Properties']['Origins']['Required'])
+        self.assertFalse(patched['ResourceTypes']['AWS::CloudFormation::WaitCondition']['Properties']['Handle']['Required'])
```

Comment:
this a repeat?

Reply:
Oops. Second one should be Origins. Will fix it in a minute

Reply:
Hey, sorry. I'm having trouble seeing what is repeated?